### PR TITLE
apps/users: make switching the language via the indicator also change

### LINKED ISSUE #2719: toggle language bugged with a set preferred languages
**URL:** https://adhocracy.plus/musterorganisation/ideas/2021-04618
**user:** any 
**expected behaviour:** the comment section is displayed in the language I chose 
**behaviour:** the comment section is always in German 
**important screensize:** any
**device & browser:** any

Notes: Having a set prefered language in the account settings will lead to some weird, partially translated pages when changing the language toggle. As you can see in the screenshot below, I changed the language to English but my prefered language is German. So the comment section stays in German but some other elements like date, refrence no. or "go back to overview" are translated in English.
 
I prefer, although it's nice to have a globally prefered language, that the language toogle on top works as expected and therefore translates every element on the page.

grafik

### PR DESCRIPTION
**Pull request by goapunk**
the users preferred language.

- add missing cookie settings to language cookie

fixes #2719

**Tasks**
- [ ] PR name contains story or task reference
- [x] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [x] Changelog

### CODE REVIEW

File: apps/users/middleware.py [hunk 1]
```diff
@@ -13,6 +13,13 @@ def __call__(self, request):
             translation.activate(request.user.language)
             if cookie != request.user.language:
                 response.set_cookie(
-                    settings.LANGUAGE_COOKIE_NAME, request.user.language
+                    settings.LANGUAGE_COOKIE_NAME,
```
Comment (goapunk) [inline apps/users/middleware.py:16]: taken from https://github.com/django/django/blob/main/django/views/i18n.py#L64C12-L73C14

**Comment by m4ra** [comment]
@goapunk this fix has a side-effect that when we navigate the website and at some point switch to a language different from our preferences, when we navigate back to our account settings, then this new language becomes as our preferred (the language input becomes this chosen language we toggled). While currently whenever we navigate back to our account settings, it switches back to our default/preferred language. Is that ok @TeaJayyy?

**Comment by m4ra** [comment]
@goapunk did a quick demo to Tietje and the preferred language change is what should be expected, so am going to merge this.

**Review by m4ra — APPROVED** [review]
thank you :)

**Pull request merged**